Homework2/task1/src/homework/Initialize.java

```diff
@@ -0,0 +1,53 @@
+package homework;
+
+import java.util.Scanner;
+
+/**
+ * Class Initialize has methods setLine and validation, that input line from keyboard,
+ * validate them and returns an array of words form the line.
+ */
+public class Initialize {
+
+    /**
+     * Method setLine input the line from keyboard, splits the words
+     * (delimiters are whitespace and comma). It returns an array of words.
+     * Also method verify that line isn't empty.
+     *
+     * @return array of words from the input line
+     */
+    public static String[] setLine() throws Exception {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Set line");
+        String line = scanner.nextLine();
+        scanner.close();
+        // If the line is empty, method throws exception
+        if (line.isEmpty()) {
+            throw new Exception("Empty line");
+        }
+        // All commas are replaced by whitespace to facilitate split words
+        line = line.replace(',', ' ');
```

Comment:
This is not valid, if you have rule that counts spaces this code will change its result. Here your program knows nothing about rules, so please rework it